tests/test_result.py

```diff
@@ -130,6 +130,14 @@ def test_unwrap_or_else() -> None:
     assert n.unwrap_or_else(str.upper) == 'NAY'
 
 
+def test_unwrap_or_raise() -> None:
+    o = Ok('yay')
+    n = Err('nay')
+    assert o.unwrap_or_raise(ValueError) == 'yay'
+    with pytest.raises(ValueError):
```

Comment:
can you add a test to verify that the exception instance actually received the error value?

💡 `pytest.raises` can give you the exception info, which has a `.value` attribute with the actual exception instance. something like this (untested):


``` python
    with pytest.raises(ValueError) as exc_info:
        ...

    exc = exc_info.value
    assert exc.args == ('nay,')
```

Reply:
Thanks, applied.